isis/src/dev/apps/m3loc2net/m3loc2net.xml

```diff
@@ -0,0 +1,144 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<application name="m3loc2net" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="http://isis.astrogeology.usgs.gov/Schemas/Application/application.xsd">
+
+  <brief>
+    Creates a control network containing ground points either using a Chandrayaan LOC file
+    or the image camera for lat/lon/radii values.
+  </brief>
+
+  <description>
+    <p>
+      This application is for development is not supported for general public use.

```

Comment:
Need an **and** between "development" and "is".

I would add ```<strong>``` tag around this sentence as well.